### DISCUSSION TITLE
AU - Content of admnUnit attribute

### DISCUSSION THREAD

**Discussion by bfrichet3** [INSPIRE-MIF/helpdesk#86] [Q&A] [answered]
Dear Sir or Madam

I am writing to youy because I have a question concerning the admUnit attribute of AdministrativeBoundary object types.

I read here that the field should be populated with "The administrative units separated by this administrative boundary."

![image](https://user-images.githubusercontent.com/80756452/163341599-3474e600-6601-46ce-bf35-e8fce9671345.png)

If the considerated AdministrativeBoundary object is a boundary separating AdministrativeUnit objects of several levels (like a boundary between 4th and 5th level AdministrativeUnit) should we make a reference to all the separated AdministrativeUnit objects?

Regards

**Comment by sMorrone** [comment, marked as answer]
Dear @bfrichet3,
yes, you should make a reference to all the separated AdministrativeUnit objects.

**Comment by bfrichet3** [comment]
thank you very much

**Comment by sMorrone** [comment]
Since the purpose of the the technical guidelines repository is the management of the change proposals to the TGs, I am moving this issue to the [general helpdesk ](https://github.com/INSPIRE-MIF/helpdesk)repository. More specifically, I will move it to the "Discussions" tab and classify it as "[Q&A](https://github.com/INSPIRE-MIF/helpdesk/discussions/categories/q-a)"
